src/version.h

```diff
@@ -9,4 +9,5 @@ std::uint32_t GetVersionInt(int major = LC0_VERSION_MAJOR,
 
 std::string GetVersionStr(int major = LC0_VERSION_MAJOR,
                           int minor = LC0_VERSION_MINOR,
-                          int patch = LC0_VERSION_PATCH);
+                          int patch = LC0_VERSION_PATCH,
+                          bool release = LC0_IS_RELEASE);
```

Comment:
Don't forget about GetVersionInt!

Reply:
Hmm on second thought, I think the LC0_IS_RELEASE should become a string so that we can insert the git sha of length 6 in case of dev mode lateron.